src/components/PositionsAudit/PositionsAudit.js

```diff
@@ -21,14 +23,14 @@ const TYPE = queryConstants.MENU_POSITIONS_AUDIT
 class PositionsAudit extends PureComponent {
   componentDidMount() {
     const {
-      loading, setTargetIds, fetchPaudit, match,
+      loading, setTargetIds, fetchPaudit, match, targetIds,
     } = this.props
-    if (loading) {
-      const id = (match.params && match.params.id) || ''
-      if (id) {
-        setTargetIds(id.split(','))
-        fetchPaudit()
-      }
+    const ids = _get(match, 'params.id', '').split(',')
+
+    const isIdChanged = !_isEqual(ids, targetIds)
```

Comment:
maybe it's better to sort arrays between comparison
![image](https://user-images.githubusercontent.com/2828226/71881666-c4a5ee00-3143-11ea-8d2a-370f0c83e2a1.png)

Reply:
array is used to support multiple ids, though in practice only 1 id is used, unless user manually changes url
that will handle possible redundant requests in the future, thanks! :)